tests/CMakeLists.txt

```diff
@@ -28,6 +28,7 @@ if(NOT ENABLE_SERVER_TESTS)
   return()
 endif()
 
+add_subdirectory( XrdOssTests )
```

Comment:
@amadio - do we have to worry about race conditions between the build and the running of the tests?

Here, `XRootD::http::test` will require the built library.  I assume that `ctest` is guaranteed to only run after a full build so we don't need any explicit dependency in CMake.

Is that assumption correct?

Reply:
Yes, `ctest` runs only after everything is already built.